### ISSUE TITLE
"..." more menu cannot be accessed from the touch monitor of a PC

### ISSUE THREAD

**Issue by Softvision-VladBacia** [mozilla/activity-stream#3280]
[Affected versions]:
- Firefox 57.0a1, Build ID: 20170828100127

[Affected Platforms]:
- All Windows

[Prerequisites]:
- browser.newtabpage.activity-stream.enabled is set to true in about:config on a clean new profile.
- have a PC that has a Touch monitor (e.g. [Dell Inspiron 2330](http://www.dell.com/us/dfh/p/inspiron-one-23-2330-aio/pd)).

[Steps to reproduce]:
1. Start the browser with the profile from prerequisites and open a New Tab.
2. Access "..." more menu for the tiles from Activity Stream page from the touch monitor.
 
[Expected results]:
- "..." more menu can be accessed for every tile from Activity Stream 

[Actual results]:
- When the tiles are tapped, the "..." more menu icon is displayed for a fraction of a second and after that the page that was tapped is loaded in the same tab. Couldn't find a way to access the "..." more menu. Long press on a tile will open the browser context menu.

[Notes]:
- Attached a screen recording of the issue:

![touchscreen](https://user-images.githubusercontent.com/9248513/29821283-04b7c4d4-8cd0-11e7-97a0-95a30c2f081a.gif)

**Comment by Mardak** [comment]
This is one of the items from #2998

**Comment by Mardak** [comment]
Design said not for 57 (Blocked label)

**Comment by Mardak** [comment]
Moved to https://bugzilla.mozilla.org/show_bug.cgi?id=1432299
